cmd/es-rollover/app/init/flags.go

```diff
@@ -23,12 +24,7 @@ const (
 // Config holds configuration for index cleaner binary.
 type Config struct {
 	app.Config
-	Shards                       int
-	Replicas                     int
-	PrioritySpanTemplate         int
-	PriorityServiceTemplate      int
-	PriorityDependenciesTemplate int
-	PrioritySamplingTemplate     int
+	cfg.Indices
```

Comment:
@yurishkuro both `Indices` and `app.Config` have `IndexPrefix` in them and for the purpose of this package, we use the `IndexPrefix` from `app.Config`. Do you have any thoughts on what we should do here? It's a bit confusing at the moment.

Reply:
Instead of embedding `cfg.Indices`, should we just add what is needed?

Reply:
does `app.Config` actually need the prefix? Is it applicable to all sub-commands?

Reply:
yeah - currently the `indexPrefix` field in `app.Config` is initialized in `InitFromViper` as 
```go
	c.IndexPrefix = v.GetString(indexPrefix)
	if c.IndexPrefix != "" {
		c.IndexPrefix += "-"
	}
```
and this is what is used in `cmd/es-rollover/app`